### PR TITLE
update Arab country codes for (KFC , McDonald's )

### PR DESCRIPTION
**Pull request by waso99**
update Arab country codes for (KFC , McDonald's )

### CODE REVIEW

**Comment by LaoshuBaby** [comment]
I draw 2 sketch to help you understand.the locationset logic we are using

For those countries haven't got a specify item, they will use global one, and those country have special item shouldn't appear in 001, or both global item and special item will affect.

IMG_20220322_013106</img> Screenshot_2022-03-22-01-29-16-261_com miui notes</img>

File: data/brands/amenity/fast_food.json [hunk 1]
```diff
@@ -1718,7 +1718,41 @@
...
+          "ae",
+          "bh",
+          "kw",
+          "qa",
+          "sa",
+          "om",
+          "lb",
+          "eg"
+        ],
+      "exclude": ["001", "cn", "es"]
```
[the remark this answers is not in the document]
Comment (waso99) [reply data/brands/amenity/fast_food.json:1735]: So, this need more work with other tags?

[the remark this answers is not in the document]
Comment (LaoshuBaby) [reply data/brands/amenity/fast_food.json]: I will recheck for other area tomorrow, thank you for maintain those arabic countries' information(Then I'll merge it)

**Comment by arch0345** [comment]
Thanks @waso99!

**Pull request merged**